daemon/algod/api/server/v1/handlers/handlers.go

```diff
@@ -603,6 +603,10 @@ func SuggestedParams(ctx lib.ReqContext, w http.ResponseWriter, r *http.Request)
 	params.GenesisHash = gh[:]
 	params.LastRound = uint64(stat.LastRound)
 	params.ConsensusVersion = string(stat.LastVersion)
+
+	proto := config.Consensus[stat.LastVersion]
```

Comment:
Consider looking up the last block and using that block's consensus params.

Reply:
I was under the impression that `node.Status()` does exactly that - or is it better to do it explicitly in the handler?

Reply:
Oops, you're right, so this does the right thing already.